examples/data-objects/shared-text/src/pageCacher.ts

```diff
@@ -11,7 +11,9 @@ interface IWindow extends Window {
 // Extract the head and body HTML. Remove any <script> tag. And then wrap inside a basic HTML page.
 export function createCacheHTML(): void {
     const [bodyHTML, headHTML] = [document.body.innerHTML, document.head.innerHTML];
+    // eslint-disable-next-line unicorn/no-unsafe-regex
```

Comment:
This seems scary - potentially exponential-time, according to this?  https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-unsafe-regex.md